### PR TITLE
Update policies

### PR DESCRIPTION
**Pull request by pietroalbini**
This PR updates the policies section as recommended by Mozilla Legal:

* The Privacy Policy was updated to version 1.1, with the following changes:
  * Added a note about crates.io using Sentry for error reporting.
  * Added a note about docs.rs storing visitor logs for a year.
* A disclaimer was added on top of all translated policies, saying the canonical version is the english one:

### CODE REVIEW

**Comment by pietroalbini** [comment]
Do not merge this yet.

**Review by Manishearth — APPROVED** [review]

**Comment by pietroalbini** [comment]
This is blocked on https://github.com/rust-lang/core-team/issues/22.

**Comment by pietroalbini** [comment]
Core Team approved!

**Pull request merged**
